packages/engine/stdlib/src/py/hstd/test_init.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from .agent import AgentState
+from .spatial import Topology
+from .init import scatter, grid, stack, create_layout
+
+init_topology = Topology([0, 2], [0, 2])
+
+agent = AgentState(agent_name="test")
+agent_function = lambda: AgentState(agent_name="test")
+
+num_agents = 4
+
+scatter_agents = scatter(num_agents, init_topology, agent)
```

Comment:
It looks like these global variables are only referenced by the `test_scatter` function. Probably nicer to move them to the function's scope.